dxr/plugins/clang/tests/test_anon_ns/test_anon_ns.py

```diff
@@ -5,13 +5,13 @@ class AnonymousNamespaceTests(DxrInstanceTestCase):
     """Tests for anonymous namespaces"""
 
     def test_function(self):
-        self.found_line_eq('+function:"(anonymous namespace in main.cpp)::foo()"',
+        self.found_line_eq('+function:"<anonymous namespace in main.cpp>::foo()"',
                            'void <b>foo</b>() /* in main */', 5)
-        self.found_line_eq('+function:"(anonymous namespace in main2.cpp)::foo()"',
+        self.found_line_eq('+function:"<anonymous namespace in main2.cpp>::foo()"',
                            'void <b>foo</b>() /* in main2 */', 5)
 
     def test_function_ref(self):
-        self.found_line_eq('+function-ref:"(anonymous namespace in main.cpp)::foo()"',
+        self.found_line_eq('+function-ref:"<anonymous namespace in main.cpp>::foo()"',
                            '<b>foo</b>();  /* calling foo in main */', 12)
-        self.found_line_eq('+function-ref:"(anonymous namespace in main2.cpp)::foo()"',
+        self.found_line_eq('+function-ref:"<anonymous namespace in main2.cpp>::foo()"',
```

Comment:
Not especially recently. You'd have to `git blame` the `vagrant_provision.sh` file to see exactly when. You might need to re-run `make` on your VM if it's still kicking out `<`; everything's supposed to emit `(`, regardless of clang version now. Here's the line where rewriting happens: https://github.com/mozilla/dxr/blob/d1398ea9d98a65cf035743f6a0215b2e1c27d878/dxr/plugins/clang/dxr-index.cpp#L276.